pipeline-maven/src/test/java/org/jenkinsci/plugins/pipeline/maven/DependencyGraphTest.java

```diff
@@ -501,8 +506,14 @@ public void verify_nbm_downstream_simple_pipeline_trigger() throws Exception {
         assertThat(upstreamCause).isNotNull();
     }
 
+    public boolean checkIsLinuxAndDockerSocketExists() {
+        return OS.current() == OS.LINUX && Files.exists(Paths.get("/var/run/docker.sock"));
```

Comment:
For reference, the idiom used in most plugins is to actually attempt to contact a Docker daemon and skip the test if this fails; for example: https://github.com/jenkinsci/ldap-plugin/blob/bd1a491714dc7e9e5c7293a52d60aa83506b2e28/src/test/java/hudson/security/docker/PlanetExpressTest.java#L32-L34